### ISSUE TITLE
Rodwell Mires and Heaths has wrong year in website refs

### ISSUE THREAD

**Issue by sacrevert** [BiologicalRecordsCentre/bsbi-atlas-drupal8#32]
Appears as 1992 (e.g. https://plantatlas2020.org/atlas/2cd4p9h.1aa) but should be 1991. I don't know exactly what the issue is here, as is ok in the book, so leaving this for @kwal2 and @japonicus to sort. This could be closed when sorted, as it is not a book erratum to catalogue.

**Comment by japonicus** [comment]
The online meta-data suggests that 1992 is correct, are you certain that this volume was 1991?
e.g.
https://isbnsearch.org/isbn/9780521391658

**Comment by sacrevert** [comment]
Yes, have the physical book here and it is 1991 (but perhaps it was reprinted in 1992?)

**Comment by japonicus** [comment]
Thanks, I'll change it. It's odd that even Cambridge University Press lists first publication as 1992, I wonder if physical publication slipped by a month (in to Jan 1992) after typesetting was completed.

**Comment by sacrevert** [comment]
@japonicus Can you close when you are happy?
